### PR TITLE
feat: implement sigil

### PR DESCRIPTION
**Pull request by izabellalarsson**
General work is done. 

TODO: 
- [x] Get new design from Johan and update
- [x] copy to clickboard
- [x] add address to a tag in sigil snippet

### CODE REVIEW

**Comment by irony** [comment]
Great! 

Since I started thinking about this feature in december I thought it could be good to just leave my thoughts here. You might have already found other better ways to do it, if so just ignore this. 

The idea was that every time a sigill is called - we check the cache and see if it's still valid- otherwise we schedule a new test (add a job to bullMQ) and serve a success badge (or previous result) while the test is running. That way we can rely on the badge as the initiator of a recurrent test without having to first implement the whole registration process.

**Comment by izabellalarsson** [comment]
Screenshot 2022-01-21 at 16 33 06

**Comment by izabellalarsson** [comment]
Screenshot 2022-01-26 at 10 38 41
Screenshot 2022-01-26 at 10 39 09

**Review by mull — APPROVED** [review]

File: packages/web/src/views/tests/show.pug [hunk 1]
```diff
@@ -31,16 +31,42 @@ mixin group_index(passed, index)
...
+
 mixin start_msg
     if (allTestsPassed)
-      h1(class="mb-6") Grattis!
-      p Alla testen passerade. Du tar verkligen säkerhet på allvar. 
-      //Grattis! Du har förtjänat vårt sigill!
-      // p Bädda in sigillet på din webbplats för att visa att din webbplats tar säkerhet på allvar. Välj mellan de två olika storlekarna och placera inbäddningskoden någonstans på din webbplats (exempelvis i sidhuvud/sidfot).
+      h1(class="mb-6") Grattis din sajt är Robust, du har förtjänat Robust-sigillet! 
+      p Bädda in sigillet på din webbplats för att visa att ni tar säkerhet på allvar. Välj mellan de två olika storlekarna och placera inbäddningskoden någonstans på din webbplats (exempelvis i sidhuvud/sidfot).
+      p <i>Obs. Detta är en beta version, en ny embedkod kommer att komma senare.</i>
```
Comment (irony) [inline packages/web/src/views/tests/show.pug:65]: Obs! Detta är en betaversion, eventuellt kan du behöva byta ut till ny embedkod senare.

**Review by irony — CHANGES_REQUESTED** [review]
En liten copyändring i övrigt toppen!

File: packages/web/src/views/tests/show.pug (hunk 1 above)
Comment (irony) [reply packages/web/src/views/tests/show.pug:65]: Vi kommer antagligen inte behöva en ny embedkod framöver så jag skrev ett förslag ovan om att den KAN ändras.

**Pull request merged**